src/services/egoTokenService.ts

```diff
@@ -0,0 +1,71 @@
+import urlJoin from "url-join";
+import memoize from "memoizee";
+import axios from "axios";
+import jwt from "jsonwebtoken";
+import ms from "ms";
+import { NextFunction, Request, Response, RequestHandler } from "express";
+import { ForbiddenError, UnauthorizedError } from "../utils/errors";
+
+const getKey = memoize(
+    async (egoURL: string) => {
+        const response = await axios.get(urlJoin(egoURL, "oauth/token/public_key"));
```

Comment:
why the other part of the url is hardcoded ?

Reply:
good point, ill remove this and chop off one dependency.

Reply:
you are using 4 libraries in one line and you shamed me for adding yaml2js .. :ok_hand: :joy:

Reply:
:nauseated_face: